src/interp/binary-reader-interp.cc

```diff
@@ -856,6 +849,16 @@ Result BinaryReaderInterp::EndFunctionBody(Index index) {
 Result BinaryReaderInterp::OnLocalDeclCount(Index count) {
   local_decl_count_ = count;
   local_count_ = 0;
+  // FIXME(Soni): does the value of `values` even matter here? it used to be
+  // always 0 when this call was in BeginFunctionBody.
```

Comment:
I'm not sure this FIXME comment is useful, maybe just remove it?

Reply:
yeah that's fair.

... ugh, the more we look at all this code the more *questionable* stuff we find, like the `func_->handlers.size()` in `BeginFunctionBody` (which, presumably, always evaluates to `0`?). but that's not the fix we came here for.